integration_test/test_videoroom/test/support/simulcast_mustang.ex

```diff
@@ -23,28 +23,15 @@ defmodule SimulcastMustang do
   @impl true
```

Comment:
Now we should rename `simulcast_mustang` to something other? Maybe `TestMustang`?